src/coreclr/tools/aot/ILCompiler/ILCompilerRootCommand.cs

```diff
@@ -265,8 +267,8 @@ public ILCompilerRootCommand(string[] args) : base(".NET Native IL Compiler")
                         // + a rsp file that should work to directly run out of the zip file
 
                         Helpers.MakeReproPackage(makeReproPath, context.ParseResult.GetValue(OutputFilePath), args, context.ParseResult,
-                            inputOptions : new[] { "r", "reference", "m", "mibc", "rdxml", "directpinvokelist", "descriptor" },
-                            outputOptions : new[] { "o", "out", "exportsfile" });
+                            inputOptions : s_inputOptions,
```

Comment:
This method is only executed once during the execution of the program. Caching the array in a static variable is deoptimization. This change should be reverted.